### ISSUE TITLE
`--run` interprets shell quotes wrong

### ISSUE THREAD

**Issue by Dominik-K** [telepresenceio/telepresence#865]
### What were you trying to do?

Trying out [Telepresence with Golang](https://www.telepresence.io/howto/golang). Quoted the argument for `--run`:

    telepresence --namespace telepresence --run "$GOPATH/bin/wuzz http://hello-world.telepresence:8000"

to avoid [falsely interpreting whitespaces inside the argument](https://mywiki.wooledge.org/Quotes).

### What did you expect to happen?

Normal run. Same as `--run` without quotes.

### What happened instead?

Telepresence interpreted the whole argument as a file.?

### Automatically included information

Command line: `['/usr/local/bin/telepresence', '--namespace', 'telepresence', '--run', '/Users/dominik/go/bin/wuzz http://hello-world.telepresence:8000/']`
Version: `0.95`
Python version: `3.7.1 (default, Nov  8 2018, 12:42:00) 
[Clang 8.0.0 (clang-800.0.42.1)]`
OS: `Darwin Dominiks-MacBook-Pro.local 18.2.0 Darwin Kernel Version 18.2.0: Fri Oct  5 19:41:49 PDT 2018; root:xnu-4903.221.2~2/RELEASE_X86_64 x86_64`
Traceback:

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 129, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 78, in main
    runner, remote_info, env, socks_port, ssh, mount_dir
  File "/usr/local/bin/telepresence/telepresence/outbound/__init__.py", line 60, in launch
    runner_, remote_info, command, args.also_proxy, env, ssh
  File "/usr/local/bin/telepresence/telepresence/outbound/local.py", line 122, in launch_vpn
    process = Popen(command, env=env)
  File "/usr/local/Cellar/python/3.7.1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 769, in __init__
    restore_signals, start_new_session)
  File "/usr/local/Cellar/python/3.7.1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 1516, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/dominik/go/bin/wuzz http://hello-world.telepresence:8000/': '/Users/dominik/go/bin/wuzz http://hello-world.telepresence:8000/'

```

**Comment by ark3** [comment]
Sorry about the crash! That was fixed in #870 and will function correctly in the next release.

Telepresence does not interpret shell quotes; that's the shell's job. Telepresence behaves the same as other command line software that executes a command on your behalf:

```shell
$ cat /etc/hosts
##
# Host Database
#
# localhost is used to configure the loopback interface
# when the system is booting.  Do not change this entry.
##
127.0.0.1       localhost
255.255.255.255 broadcasthost
::1             localhost

$ env "cat /etc/hosts"
env: cat /etc/hosts: No such file or directory

$ sudo "cat /etc/hosts"
sudo: cat /etc/hosts: command not found

$ docker run --rm -it alpine "cat /etc/hosts"
docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"cat /etc/hosts\": stat cat /etc/hosts: no such file or directory": unknown.

$ 
```

**Comment by Dominik-K** [comment]
Tested it again and found out that Telepresence assumes `--run` is the last argument. The `telepresence --help` suggests it's a normal flag:

    usage: telepresence [SKIPPED]
                        [--run-shell | --run ... | --docker-run ...]

I suggest using `[COMMAND] [ARG]` as positional arguments, like in `docker run`:

    Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

or even better explicitly break them from beforehand arguments with `--`, like in `kubectl`:

    Usage:
       kubectl run NAME --image=image [SKIPPED] -- [COMMAND] [args...] [options]
